---
name: Biology
description: TBD
slug: /biology
references:
  - number: 1
    author: MacFarlane, L., and Murphy, P, R.
    title: "MicroRNA: Biogenesis, Function and Role in Cancer"
    journal: Curr Genomics.
    volume: "11(7): 537-561"
    published_date: "2010"
    doi: 10.2174/138920210793175895
  - number: 2
    author: Wang, J., Chen, J., and Sen, S.
    title: microRNA as Biomarkers and Diagnostics.
    journal: Journal of Cellular Physiology
    volume: "231(1): 25-30"
    published_date: 2022
    doi: 10.1002/jcp.25056	
  - number: 3
    author: An, X., Quan, H., Lv, J., Meng, L., Wang C., Yu, Z., et al.
    title: Serum microRNA as potential biomarker to detect breast atypical hyperplasia and early-stage breast cancer.
    journal: Future Oncology
    volume: "14(30): 3145-3161"
    published_date: Dec. 2018
    doi: 10.2217/fon-2018-0334
  - number: 4
    author: Mar-Aguilar, F., Mendoza-Ramírez, J, A., Malagón-Santiago, I., Espino-Silva, P, K., Santuario-Facio, S, K., Ruiz-Flores, P., Rodríguez-Padilla, C., and Reséndex-Pérez, D.
    title: Serum Circulating microRNA Profiling for Identification of Potential Breast Cancer Biomarkers
    journal: Disease Markers
    volume: "34(3): 163-169"
    published_date: Mar. 2013
    doi: 10.3233/DMA-120957
  - number: 5
    author: Müller, V., et al
    title: Changes in serum levels of miR-21, miR-210, and miR-373 in HER2-positive breast cancer patients undergoing neoadjuvant therapy: a translational research project within the Geparquinto trial
    journal: Breast Cancer Research and Treatment
    volume: "147(1), 61-68"
    published_date: Aug. 2014
    doi: 10.1007/s10549-014-3079-3
  - number: 6
    author:  Butler, A, E., et al.
    title: microRNA Expression in Women With and Without Polycystic Ovarian Syndrome Matched for Body Mass Index
    journal: Frontiers in Endocrinology
    volume: "11"
    published_date: Apr. 2020
    doi: 10.3389/fendo.2020.00206
  - number: 7
    author: Cirillo, F., et al.
    title: miRNAs Regulating Insulin Sensitivity Are Dysregulated in Polycystic Ovary Syndrome (PCOS) Ovaries and Are Associated With Markers of Inflammation and Insulin Sensitivity
    journal: Frontiers in Endocrinology
    volume: "10"
    published_date: Dec. 2019
    doi: 10.3389/fendo.2019.00879
  - number: 8
    author: Motahari Rad, H., et al.
    title: Characterization of altered microRNAs related to different phenotypes of polycystic ovarian syndrome (PCOS) in serum, follicular fluid, and cumulus cells
    journal: Taiwanese Journal of Obstetrics and Gynecology
    volume: "61(5), 768–779"
    published_date: Sept. 2022
    doi: 10.1016/j.tjog.2022.05.013
---

<h1>Biology</h1>
<h2 style={{ paddingTop: 4 }}>Project description</h2>
The project this year has focused on combating the antiquated tests currently in place for diseases
linked to women’s health. Our method involves multiplexed regulation which uses four stacked toehold
switches, for the four diseases, which each test for three miRNA. The switches incorporate genetic
'AND' gates, allowing for high specificity. Uniquely, our project allows for the miRNA binding sites
to be swapped meaning various diseases can be tested for, using the database we have compiled.

The presence of the miRNA can be tested simultaneously using one blood sample, with the turnaround
time for results being significantly improved compared to older tests. The toehold switches being
used will synthesise different fluorescent proteins, once they have unfolded, given the presence of
the correct miRNA. The chosen fluorescent proteins have varying wavelengths so can be tested
separately, as shown on our <a href="#" onClick={() => navigate("/hardware/")}>
Hardware Page
</a>.

To increase the accessibility of our test, we have chosen to use miRPA to amplify the miRNA taken in
our sample. This allows the miRNA to be isothermally amplified, and therefore doesn’t require the use
of an expensive thermocycler. More information on the probes used can be found on our <a href="#" onClick={() => navigate("/software/")}>Software Page</a>.<h2 style={{ paddingTop: 8 }}>MicroRNA</h2>
Our project uses microRNA biomarkers to indicate the presence of disease. MicroRNAs (miRNAs) are
small, highly conserved non-coding RNA molecules involved in the regulation of gene expression

<u>ref1</u>. These microRNAs circulate in the blood, and can be shown to be either upregulated, or
downregulated in certain conditions. The dysregulation of these diseases can be linked to
developmental stages of diseases <u>ref2</u> and by using toehold switches, which have binding sites
for the upregulated miRNA we can detect the disease. While our system is meant to be modular, we
have chosen endometriosis, PCOS, ovarian cancer and breast cancer to prove the theory. Given we
needed 3 miRNA for each disease we have chosen the following, which are all upregulated:
<h3 style={{ paddingTop: 4 }}>Breast Cancer:</h3>
<ul>
	<li>
		miRNA-24-3p: can be used for early detection
		<u>ref3</u>
	</li>
	<li>
		miRNA-21-3p: has different expression patterns in serum of breast cancer
		<u>ref4</u>
	</li>
	<li>
		miRNA-373-5p
		<u>ref5</u>
	</li>
</ul>
<h3 style={{ paddingTop: 4 }}>PCOS:</h3>
<ul>
	<li>
		miRNA-19b-3p
		<u>ref6</u>
	</li>
	<li>
		miRNA-19b-3p
		<u>ref7</u>
	</li>
	<li>
		miRNA-1260a
		<u>ref8</u>
	</li>
</ul>
<h3 style={{ paddingTop: 4 }}>Ovarian Cancer:</h3>
<ul>
	<li>
		miRNA-320a-5p
		<u>ref9</u>
	</li>
	<li>
		miRNA-320b
		<u>ref10</u>
	</li>
	<li>
		miRNA-4640-5p
		<u>ref11</u>
	</li>
</ul>
<h3 style={{ paddingTop: 4 }}>Endometriosis:</h3>
<ul>
	<li>
		miRNA-145-5p
		<u>ref12&13</u>
	</li>
	<li>
		miRNA-199a-3p
		<u>ref12&13</u>
	</li>
	<li>
		miRNA-125b-5p
		<u>ref12&13</u>
	</li>
</ul>
<h2 style={{ paddingTop: 8 }}>Toehold Switches</h2>
Toehold switches are synthetic RNAs which are able to use RNA fragments to bring about transcription
of certain proteins, mimicking the role of a messenger RNA from within a normal cell. In its OFF
state, the toehold switch remains in a hairpin structure with an 'input' recognition site for the
RNA, and a hidden or inaccessible ribosome binding site. This allows for the synthesis of proteins
under exact biological conditions and therefore allows for the detection of specific or complex
environments
<u>ref14</u>
Within Genoswitch, we use microRNA as a biomarker to indicate when a specific blood environment,
brought about by a certain disease, occurs
<u>ref15</u>. When the miRNA is sufficiently upregulated, the ribosome binding site for the various fluorescent
proteins is unfolded, decreasing the length of the stem and allowing for exposure of the start codon,
thus allowing for translation.

<h2 style={{ paddingTop: 8 }}>miRPA</h2>
As described by project RIBOTOX, it is difficult to discriminate between leaky expression of toehold
switches and <u>less than</u>2.25M concentrations of miRNAs <u>ref16</u>. We therefore needed to amplify
the miRNA. We have chosen to use miRPA, similarly to the RIBOTOX project. It is a single tube, isothermal
alternative to PCR so both increases accessibility to groups with less access to thermocyclers and also
is easier to perform amplification on miRNA than PCR.

We therefore needed to amplify the miRNA. We have chosen to use miRPA, similarly to the RIBOTOX project. It is a single tube, isothermal alternative to PCR so both increases accessibility to groups with less access to thermocyclers and also is easier to perform amplification on miRNA than PCR.

Two DNA probes, one with 5' phosphorylation, bind to the miRNA, and are ligated together by DNA ligase.
Then, primers are added with DNA polymerase, and complementary strands to the ligated probes are synthesised.
Then, RPA can take place: primers, associated with recombinase protein dislodge the strands, replicating them in a similar method to PCR but as no heat cycles are required to break up the strands, the process can take place isothermally.

In order for the miRNA to be detected, we will need to use 'asymmetric RPA': an excess of forward primers are added (usually 5x the amount), so an excess of the strand that was originally miRNA form, so there is now ssDNA with the miRNA code in DNA.
This can be detected by toehold switches.
In order to design probes for miRPA, we can use NUPACK's design functions in its API to find probes which can bind to the miRNAs, but have overhangs which do not bind within themselves, to ensure primers can easily anneal to them.
